### DISCUSSION TITLE
How to retrieve business level stats?

### DISCUSSION THREAD

**Discussion by mrveera** [ory/hydra#2444] [Q&A] [answered]
We have a use where we need to find number of uniq user logins and number of logins per day, week, month etc. Is that info available by any chance already?

**Comment by vinckr** [comment, marked as answer]
I dont think it is collected directly in Hydra, but you could track it in your login node?

**Reply by vinckr** [reply]
Hey @veera83372,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝

**Reply by mrveera** [reply]
Hi @vinckr 

Thanks for checking, we actually went on a different route get those stats, Somewhere I feel hydra could do that in better way  because hydra know about all logins and logouts etc. anyways it is very use case specific . let this one be in answered state
